.github/workflows/perl-cpan-libraries.yml

```diff
@@ -310,6 +318,7 @@ jobs:
         image: [packaging-plugins-bullseye, packaging-plugins-bookworm, packaging-plugins-jammy, packaging-plugins-bullseye-arm64]
         name:
           [
+            "Paws",
```

Comment:
can you restore Paws using alphabetical order

Reply:
We've decided to put it first because it's so long (more than 2 hours) that we wanted it to be the first to run.

Reply:
the first of the list is not especially the first to run